.github/workflows/build_workflow.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           bash tests/test.sh
 
   publish-docs:
-    needs: [check-jobs-to-skip, pre-commit-hooks, build]
+    needs: check-jobs-to-skip
```

Comment:
Need to update this to allow publishing docs even though `pre-commit-hooks` and `build` can be skipped